app/contact/views.py

```diff
@@ -39,9 +39,9 @@ def contact_form(request):
             recipients = settings.CONTACT_FORM_RECIPIENTS.split(',')
 
             # Send it out.
-            success = email_send(subject='PPM Contact Form Inquiry',
+            success = email_send(subject='Hypatio Contact Form Inquiry',
```

Comment:
Can you change this to DBMI Data Portal instead of Hypatio?

Reply:
Derp -- right, will do.